libs/kml/serdes_geojson.cpp

```diff
@@ -264,4 +285,132 @@ void DeserializerGeoJson::Deserialize(std::string_view content)
   }
 }
 
+std::vector<std::vector<double>> convertPoints2GeoJsonCoords(std::vector<geometry::PointWithAltitude> points)
+{
+  std::vector<std::vector<double>> pairs;
+  pairs.resize(points.size());
+  for (size_t i = 0; i < points.size(); i++)
+  {
+    auto const & p = points[i];
+    auto const [lat, lon] = mercator::ToLatLon(p);
+    pairs[i] = std::vector<double>{lon, lat};
+  }
+  return pairs;
+}
+
+void GeoJsonWriter::Write(FileData const & fileData, bool minimize_output)
+{
+  using namespace geojson;
+
+  // Convert FileData to GeoJsonData and then let Glaze generate Json from it.
+  std::vector<GeoJsonFeature> geoJsonFeatures;
+
+  // Convert Bookmarks
+  for (BookmarkData const & bookmark : fileData.m_bookmarksData)
+  {
+    auto const [lat, lon] = mercator::ToLatLon(bookmark.m_point);
+    JsonTMap bookmarkProperties{{"name", GetDefaultStr(bookmark.m_name)},
+                                {"marker-color", ToCssColor(bookmark.m_color)}};
+    if (!bookmark.m_description.empty())
+      bookmarkProperties["description"] = GetDefaultStr(bookmark.m_description);
+
+    // Add '_umap_options' if needed.
+    if (auto const umapOptionsPair = bookmark.m_properties.find("_umap_options");
+        umapOptionsPair != bookmark.m_properties.end())
+    {
+      JsonTMap umap_options_obj;
+      if (auto error = glz::read_json(umap_options_obj, umapOptionsPair->second))
+      {
+        // Some error happened!
+        std::string err = glz::format_error(error, umapOptionsPair->second);
+        LOG(LWARNING, ("Error parsing '_umap_options' from KML properties:", err));
```

Comment:
LERROR will fail in debug builds, so we can check why the parser doesn't work. Does it make sense?
```suggestion
        LOG(LERROR, ("Error parsing '_umap_options' from KML properties:", err));
```